index.js

```diff
@@ -1658,6 +1665,31 @@
     return Functor.methods.map(functor)(f);
   }
 
+  //# flip :: Functor f => f (a -> b) -> a -> f b
+  //.
+  //. Apply a function contained in a Functor to a value.
```

Comment:
This sentence should begin “Applies…” rather than “Apply…” for consistency with other descriptions. Also, “a function” is misleading as there could be any number of functions. I suggest:

_Returns the result of mapping over the given functions, applying each to the given value._

Reply:
We could omit *Returns the result of* from any such description, and in this case simplify to:

*Maps over the given functions, applying each to the given value.*

The fact that the results of that is returned should be clear. What do you think?

Reply:
Sounds good to me. :)